### PR TITLE
Moves SDPA backward helper native function to functionsmanual.cpp

### PR DESCRIPTION
**Pull request by drisspg**
## Summary
chunk_grad_outputs should have been created within functionsmanual.cpp to begin with. This removes it as a native function and adds to its appropriate home. 

cc @albanD @mruberry @jbschlosser @walterddr @saketh-are

### CODE REVIEW

**Review by cpuhrsch — CHANGES_REQUESTED** [review]
Not sure this is worth it

**Review by cpuhrsch — APPROVED** [review]
but it's up to you

**Review by albanD — APPROVED** [review]

File: aten/src/ATen/native/transformers/cuda/attention.cu [hunk 1]
```diff
@@ -1042,5 +1025,24 @@ Tensor triton_scaled_dot_attention(const Tensor& q, const Tensor& k, const Tenso
 
 REGISTER_CUDA_DISPATCH(_fused_sdp_choice_stub, &_fused_sdp_choice_cuda);
 
+// !!This function is deprecated. See FunctionsManual.cpp for the implementation!!
```
Comment (albanD) [inline aten/src/ATen/native/transformers/cuda/attention.cu:1028]: This could be TORCH_CHECK(false, "..."); ?

**Review by albanD — APPROVED** [review]
Sure!

File: aten/src/ATen/native/transformers/cuda/attention.cu (hunk 1 above)
Comment (drisspg) [reply aten/src/ATen/native/transformers/cuda/attention.cu:1028]: I am afraid that if the native op is still baked into some models and this those models and if this version of libtorch is  loaded than the TORCH_CHECK would get called

**Pull request closed without merging**